### DISCUSSION TITLE
GitHub Desk Desktop

### DISCUSSION THREAD

**Discussion by dj-580U** [community/community#21612] [New to GitHub]
<p>I have created 2 git organizations and have repo’s in each. 1 is for git classroom the other is personal. My (owner) account in each has a different name &amp; email.  I have installed GitHub desktop and it only shows me my personal organization/account, but I cannot switch to the org/account for git classroom. How can I either:</p>
<ol>
<li>have a desktop icon for each org/account or</li>
<li>switch within the github desktop display to the other org. </li>
</ol>
<p>PS. I am using Windows.</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/dj-580u">@dj-580u</a> good question!</p>
<p>&gt;  I have installed GitHub desktop and it only shows me my personal organization/account, but I cannot switch to the org/account for git classroom.</p>
<p>When you log in to your GitHub account in GitHub Desktop it grants you access to all of the repositories that you have been granted access to , whether it be in your personal account or in an organization that you belong to. You can see all the repositories available for cloning by going to the menu in GitHub Desktop and selecting <code>File</code> &gt; <code>Clone Repository</code> &gt; <code>GitHub.com</code>.</p>
<p>If you are using multiple GitHub user accounts then you will need to switch accounts in GitHub Desktop from the menu (<code>File</code> &gt; <code>Options</code> &gt; <code>Accounts</code>). GitHub Desktop does not support working with multiple GitHub user accounts at this time.</p>
<p>I hope that helps explain things!</p>

**Comment by dj-580U** [comment]
<p>Thanks. I finally figured out what URL is needed FIle&gt;options&gt; after I signed out from my personal account, so I can sign in to my other org.</p>
<p>Suggestion: it would be nice if the “repository &gt;&gt;settings” list showed my organization name, userid and server info with labels. Instead of just my repo url. For example:</p>
<p>Current settings:</p>
<p>Organization - myorg</p>
<p>ID - myid</p>
<p>Server url - <a href="https://git.com/" rel="nofollow noopener">https://git.com/</a>…</p>
<p>When I 1st got started, I kept getting my ID, org and repo names mixed up.</p>

**Comment by steveward** [comment]
<p><a class="mention" href="/dj-580u">@dj-580u</a> thanks for the feedback! I’ll make a note of this for the GitHub Desktop team. Let me know if you have any other questions.</p>

**Comment by stevenroussey-privicy** [comment]
<p>There is no “options” on the file menu on a mac… </p>

**Comment by geniushourwebsite** [comment]
<p>If you’re on a mac, click “<strong>GitHub desktop</strong>” (it’s next to the files button). Then, select preferences.</p>
